enterprise-ws-spec/src/main/asciidoc/Overview.adoc

```diff
@@ -40,96 +40,98 @@ with the following characteristics:
 
 * A service implementation implements the methods of an interface that
 is describable by WSDL. The methods are implemented using a
-Stateless/Singleton Session EJB or JAX-RPC/JAX-WS web component.
+Stateless/Singleton Session EJB or Jakarta XML Web Services web component.
 * A Web service may have its interface published in one or more
 registries for Web services during deployment.
 * A Web Service implementation, which uses only the functionality
 described by this specification, can be deployed in any Web Services for
-Java EE compliant application server.
+Jakarta EE compliant application server.
 * A service instance, called a Port, is created and managed by a
 container.
 * Run-time service requirements, such as security attributes, are
 separate from the service implementation. Tools can define these
 requirements during assembly or deployment.
 * A container mediates access to the service.
 
-JAX-RPC and JAX-WS define a programming model mapping of a WSDL document
+Jakarta XML Web Services defines a programming model mapping of a WSDL document
 to Java which provides a factory (Service) for selecting which
 aggregated Port a client wishes to use. See link:#anchor-5[Figure 2] for
 a logical diagram. In general, the transport, encoding, and address of
 the Port are transparent to the client. The client only needs to make
-method calls on the Service Endpoint Interface, as defined by JAX-RPC or
-JAX-WS, (i.e. PortType) to access the service. See Chapter
+method calls on the Service Endpoint Interface, as defined by
+Jakarta XML Web Services, (i.e. PortType) to access the service. See Chapter
 link:#anchor-6[4] for more details.
 
 image:2.png[image,width=277,height=135]
 
 * [#anchor-5]##Figure 2 Client view
```

Comment:
html shows this as '[anchor-5]#Figure 2 Client view' which does not look correct